<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecordWidget</class>
 <widget class="QWidget" name="RecordWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true">/*color: rgb(255, 255, 255);*/</string>
     </property>
     <property name="title">
      <string>打卡记录</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="maximumSize">
           <size>
            <width>63</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>学号</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditID">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 0, 0);
font: 9pt &quot;宋体&quot;;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="maximumSize">
           <size>
            <width>63</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>日期</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="dateEditTime">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 0, 0);
font: 9pt &quot;宋体&quot;;</string>
          </property>
          <property name="displayFormat">
           <string>yyyy-MM-dd</string>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
          <property name="date">
           <date>
            <year>2021</year>
            <month>3</month>
            <day>8</day>
           </date>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_search">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 0, 0);</string>
          </property>
          <property name="text">
           <string>查询</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableView" name="tableView">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 0, 0);
font: 9pt &quot;宋体&quot;;</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonExport">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 0, 0);</string>
          </property>
          <property name="text">
           <string>导出</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
